app/src/main/res/layout/fragment_pocket_video.xml

```diff
@@ -70,9 +70,9 @@
         tools:ignore="ContentDescription"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginStart="400dp"
+        android:layout_marginStart="830dp"
         android:focusable="true"
-        android:background="@drawable/pocket_video_mega_tile_background"
+        style="@style/NavigationButton"
```

Comment:
Nit: It looks like this style sets `layout_height` and `layout_width`.  Can we remove these from the ImageView to avoid confusion?